### DISCUSSION TITLE
How to filter by confidence score on the tracking and detection notebook

### DISCUSSION THREAD

**Discussion by AT9991** [roboflow/notebooks#135] [Q&A]
Hi, I am playing with the Candy counting notebook. I wonder if it is possible to detect, track, and count only those candies whose confidence score is say > 0.7. Is it possible to do so?

**Comment by SkalskiP** [comment]
Hi @AT9991 👋🏻! Sorry for the late response. I was a bit busy. Yup, you can filter detections by confidence. Try this:

```python
detections = detections[detections.confidence > 0.7]
```

You can read more about advanced detection filtering [here](https://roboflow.github.io/supervision/quickstart/detections/).
